packages/terra-action-header/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 2.18.0 - (July 9, 2019)
 ------------------
 ### Changed
+* Updated to interface with react-intl's `injectIntl` removing `react's` legacy `contextTypes`.
```

Comment:
How about saying something like - "Updated components to interface with `react-intl` via its own `Intl` context instead of the old React context API"?

Reply:
Also, move this under the 'unreleased' section wherever applicable.